.eslintrc.json

```diff
@@ -0,0 +1,119 @@
+{
+  "root": true,
+  "env": {
+    "node": true
+  },
+  "parser": "@typescript-eslint/parser",
+  "plugins": [
+    "@typescript-eslint"
+  ],
+  "extends": [
+    "eslint:recommended",
+    "plugin:@typescript-eslint/recommended"
+  ],
```

Comment:
I am missing prettier. It worked really well for me in the past. Is it included in some of these recommended packages?

Reply:
It's on TS files so eslint works really well.